grammars/go.cson

```diff
@@ -162,19 +169,37 @@
         "match": "(\\+|&|\\+=|&=|&&|==|!=|\\-|\\||\\-=|\\|=|\\|\\||<|<=|\\*|\\^|\\*=|\\^=|<\\-|>|>=|/|<<|/=|<<=|\\+\\+|=|:=|;|%|>>|%=|>>=|\\-\\-|!|\\.\\.\\.|&\\^|&\\^=)"
       }
       {
-        "name": "keyword.operator.bracket.go",
-        "match": "(\\(|\\)|\\[|\\]|\\{|\\})"
+        'match': '\\{|\\}'
+        'name': 'meta.brace.curly.go'
       }
       {
-        "name": "keyword.operator.punctuation.go",
-        "match": "(\\.|,|:)"
+        'match': '\\(|\\)'
+        'name': 'meta.brace.round.go'
+      }
+      {
+        'match': '\\[|\\]'
+        'name': 'meta.brace.square.go'
       }
     ]
   'variables':
     'patterns': [
       {
-        'name': 'variable.go'
-        'match': '\\b[\\w_][\\w\\d_]*\\b',
+        'match': '\\b[\\w_][\\w\\d_]*\\b'
```

Comment:
Why would you remove the `variable.go` scope here? I would make sense to catch the package name so it doesn't get scopes as a variable, but for the others this is perfectly correct.